workspaces/resource-optimization/plugins/resource-optimization-common/README.md

```diff
@@ -1,5 +1,7 @@
-# @backstage-community/plugin-resource-optimization-common
+# Resource Optimization Common
 
-Welcome to the common package for the resource-optimization plugin!
+Shared isomorphic code for the resource-optimization plugin.
 
-_This plugin was created through the Backstage CLI_
+## Links
+
+- [Resource Optimization front-end plugin](../resource-optimization/README.md)
```

Comment:
@jkilzi any particular reason for mentioning only frontend plugin here?

Reply:
Yes, that's because the explanation about the plugin happens there.

Reply:
@jkilzi okay got it. In that case can you please share why we are not mentioning about backend-plugin here and only about frontend-plugin for my understanding. Thanks!

Reply:
The main reason is because this package is meant to be used internally by the plugin. It is even budled inside the FE plugin when deploying the lugin as a dynamic plugin.

Reply:
Can we mark it as resolved?

Reply:
Thanks for sharing the details @jkilzi